gamutils/src/test/java/com/genexus/gam/utils/test/CertificateTest.java

```diff
@@ -0,0 +1,80 @@
+package com.genexus.gam.utils.test;
+
+import com.genexus.gam.utils.CertificateUtil;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.security.cert.X509Certificate;
+
+public class CertificateTest {
+
+	private static String resources;
```

Comment:
This var is only used in setUp method, it's better to declare it there.

Reply:
Fixed in commit d1198982aeb9dff8add2e4c33cebe110b5a4301d 
Fix Quality issues and Log